metabase/metabase
# Stacked tooltip fixes

### Description

This PR contains two small changes on the stacked charts tooltip.
1) [Sort](https://metaboat.slack.com/archives/C04D0E4837E/p1677622711341419) stacked tooltip rows
2) Update [border color](https://metaboat.slack.com/archives/C04D0E4837E/p1677692387420999?thread_ts=1677622711.341419&cid=C04D0E4837E)

### How to verify

1. New question -> Sample Dataset -> Orders: Count by Created at and Products[Category]
2. Make it a stacked bar chart
3. Hover the tooltip, ensure the rows are sorted from high to low

### Demo

<img width="266" alt="Screen Shot 2023-03-01 at 8 05 23 PM" src="https://user-images.githubusercontent.com/14301985/222286936-fa84db86-87a4-4515-b760-3e12c3bac19e.png">

### Checklist
